pyspcwebgw/__init__.py

```diff
@@ -80,41 +81,45 @@ async def change_mode(self, area, new_mode):
             raise TypeError("new_mode must be an AreaMode")
 
         AREA_MODE_COMMAND_MAP = {
-            AreaMode.UNSET: 'unset',
-            AreaMode.PART_SET_A: 'set_a',
-            AreaMode.PART_SET_B: 'set_b',
-            AreaMode.FULL_SET: 'set'
+            AreaMode.UNSET: "unset",
+            AreaMode.PART_SET_A: "set_a",
+            AreaMode.PART_SET_B: "set_b",
+            AreaMode.FULL_SET: "set",
         }
         if isinstance(area, Area):
             area_id = area.id
         else:
             area_id = area
 
-        url = urljoin(self._api_url, "spc/area/{area_id}/{command}".format(
-            area_id=area_id, command=AREA_MODE_COMMAND_MAP[new_mode]))
+        url = urljoin(
+            self._api_url,
+            "spc/area/{area_id}/{command}".format(
+                area_id=area_id, command=AREA_MODE_COMMAND_MAP[new_mode]
+            ),
```

Comment:
should this comma sign be here?
98                 ),

Reply:
Same here, also ok.